web/packages/teleterm/src/ui/components/CliCommand.tsx

```diff
@@ -24,16 +24,19 @@ import { fade } from 'design/theme/utils/colorManipulator';
 
 interface CliCommandProps {
   cliCommand: string;
-  onButtonClick(): void;
-  isLoading: boolean;
-  buttonText?: string;
+  button?: {
+    text?: string;
+    onClick(): void;
+  };
```

Comment:
@gzdunek I'm always unsure about this style of props since most of the time it means that the component will receive a prop with a new identity on every render. It doesn't matter much here though and I guess it'll change with React Compiler anyway, but I'm just not sure if this style of props isn't something that's frowned upon.

Reply:
I know this style isn't super popular, but IMO it's fine most of the time (unless we need to memoize the component).
The alternative is to add a prop like `hideButton` which is more verbose.